Elevel/Elevel.Infrastructure.Services/Implementation/UserService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using Elevel.Application.Interfaces;
+using Elevel.Domain.Authentication;
+using Elevel.Domain.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Elevel.Infrastructure.Services.Implementation
+{
+    public class UserService : IUserService
+    {
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly RoleManager<IdentityRole> _roleManager;
+        private readonly TokenConfiguration _jwt;
+        public UserService(UserManager<ApplicationUser> userManager, RoleManager<IdentityRole> roleManager, IOptions<TokenConfiguration> jwt)
+        {
+            _userManager = userManager;
+            _roleManager = roleManager;
+            _jwt = jwt.Value;
+        }
+
+        public async Task<AuthenticationModel> GetTokenAsync(TokenRequestModel model)
+        {
+            var authenticationModel = new AuthenticationModel();
+            var user = await _userManager.FindByEmailAsync(model.Email);
+            if (user == null)
+            {
+                authenticationModel.IsAuthenticated = false;
```

Comment:
you can set this value as false at initialization (row 31) but I think it false by default

Reply:
Done.